src/applications/discharge-wizard/helpers/index.jsx

```diff
@@ -263,12 +263,15 @@ export const deriveIsAirForceAFRBAPortal = formValues =>
 
 // v2 Helpers
 
+// Changes Marine Corps discharge board to Navy for naming on results pages.
```

Comment:
If you have it, I'd add a little more context as to why this is needed (i.e. "Marine Corps doesn't have its own Discharge Board").